ERR_MODULE_NOT_FOUND on 2.0.0 in ESM build

I get the following error when building with v2.0.0 in an ESM environment. Build works fine in with v1.5.1

```
> NODE_ENV=production vite build

failed to load config from /home/user/development/packages/header/vite.config.js
error during build:
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/user/development/packages/header/node_modules/vite-plugin-css-injected-by-js/dist/esm/utils' imported from /home/user/development/packages/header/node_modules/vite-plugin-css-injected-by-js/dist/esm/index.js
    at new NodeError (node:internal/errors:372:5)
    at finalizeResolution (node:internal/modules/esm/resolve:437:11)
    at moduleResolve (node:internal/modules/esm/resolve:1009:10)
    at defaultResolve (node:internal/modules/esm/resolve:1218:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
    at link (node:internal/modules/esm/module_job:78:36)
```

Likely this import should end with '.js' to be esm compatible: `import { buildCSSInjectionCode, removeLinkStyleSheets } from './utils';`

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "vite-plugin-css-injected-by-js",
-  "version": "2.0.0",
+  "version": "2.0.1",
   "description": "A Vite plugin that takes the CSS and adds it to the page through the JS. For those who want a single JS file.",
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,6 +1,6 @@
 import { Plugin } from 'vite';
 import { OutputAsset, OutputChunk } from 'rollup';
-import { buildCSSInjectionCode, removeLinkStyleSheets } from './utils';
+import { buildCSSInjectionCode, removeLinkStyleSheets } from './utils.js';
 
 /**
  * Inject the CSS compiled with JS.

